backend/src/models/MeetingModel.ts

```diff
@@ -0,0 +1,53 @@
+import mongoose from 'mongoose';
+import { Schema } from 'mongoose';
+
+export interface MeetingModel {
+    _id: string;
+    name: string;
+    creator: string;
+    time: Date;
+    invited: Map<string, string[]>;
+    where: string;
+    attendance: string;
+    description: string;
+}
+
+const meetingSchema = new mongoose.Schema<MeetingModel>({
+    _id: {
+        type: String,
+        required: [true, "You must specify a meeting's id"],
+    },
+    name: {
+        type: String,
+        required: [true, "You must specify the meeting's name"],
+        trim: true,
+    },
+    creator: {
+        type: String,
+        required: [true, "You must specify the creator's id"],
+    },
```

Comment:
To make this reference the `UserModel`, you need to change the type to: `{ type: String, ref: 'User' }`.

You'll also need to do this for `attendance` but with an `Schema.Types.ObjectId` type and an `Attendance` ref